packages/browser-sdk/src/client.ts

```diff
@@ -235,10 +229,32 @@ export interface InitOptions {
   appBaseUrl?: string;
 
   /**
-   * @deprecated
-   * Use `sseBaseUrl` instead.
+   * Feature keys for which `isEnabled` should fallback to true
+   * if SDK fails to fetch features from Bucket servers. If a record
+   * is supplied instead of array, the values of each key represent the
+   * configuration values and `isEnabled` is assume `true`.
+   */
+  fallbackFeatures?: string[] | Record<string, FallbackFeatureOverride>;
```

Comment:
Let's get this merged into `features` too? 
e.g. 
```ts 
{ key: "my-foo-feature", fallback: true | false }
```

Reply:
sure, but I will do it separately. Still need to figure out how the fallback stuff is going to work, potentially syncing it down from bucket.co.